public/app/features/browse-dashboards/components/TypeCell.tsx

```diff
@@ -20,7 +21,7 @@ export function TypeCell({ row: { original: data } }: CellProps<DashboardsTreeIt
         <div className={styles.container}>
           <Icon name={iconName} />
           <Span variant="body" color="secondary" truncate>
-            Dashboard
+            <Trans i18nKey="type-cell.types.dashboard">Dashboard</Trans>
```

Comment:
Similarly here, start it with `browse-dashboards` and remove `types` from the middle